tmva/tmva/test/DNN/TestRegressionMethodDL.h

```diff
@@ -0,0 +1,148 @@
+// @(#)root/tmva/tmva/cnn:$Id$
```

Comment:
We don't really need this file. This is the implementation of the test and since it lacks any templates should be in the cxx file.

Also, to make sure the test runs in the automatic test system it should be added to the CMakeLists.txt file int the tmva/tmva/test/cnn folder.

Sorry for missing these 2 point in my first browse through.

Reply:
No problem. Will fix them